src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -32,7 +31,8 @@
     private static final String VALID_ADDRESS = BENSON.getAddress().toString();
     private static final String VALID_TELEGRAM = BENSON.getTelegram().toString();
     private static final String VALID_GITHUB = BENSON.getGithub().toString();
-    private static final Schedule VALID_SCHEDULE = BENSON.getSchedule();
+    private static final JsonAdaptedSchedule VALID_SCHEDULE = new JsonAdaptedSchedule(
```

Comment:
This line looks like it can use constructor that you mentioned is not for developers.

Reply:
Oh I was actually referring to not meant for use in main. At the end of the day it still needs to be tested. I think I will settle that when I start adding the tests, after AddEvent tests are done.